docs/api/make-docs.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+set -e
+
+if [ "$(basename $(pwd))" != "api" ]; then
+    echo "Error: you must run this from the docs/api directory" >&2
+    exit 1
+fi
+
+rm -rf api
+
+docsify init --local api
+
+rm -f api/README.md
+
+if [ -n "$NPM_PACKAGES" ]; then
+    npm_dir="$NPM_PACKAGES/lib/node_modules"
+elif [ -n "$NODE_PATH" ]; then
+    npm_dir="$NODE_PATH"
+elif [ -d "$HOME/node_modules" ]; then
+    npm_dir="$HOME/node_modules"
+elif [ -d "/usr/local/lib/node_modules" ]; then
+    npm_dir="/usr/local/lib/node_modules"
+else
+    echo "Can't determine your node_modules path; set NPM_PACKAGES or NODE_PATH appropriately" >&2
+    exit 1
+fi
+
+cp $npm_dir/docsify/node_modules/prismjs/components/prism-{json,python}.min.js api/vendor
+
+./api-to-markdown.py "$@"
+
+perl -ni.bak -e '
```

Comment:
may be nicer to use jinja2 templates to do this step, that would also let you do it all in that python script